packages/client/src/components/Footer/FooterStyle.css

```diff
@@ -0,0 +1,70 @@
+.main-footer {
+  width: 1512px;
+  height: 452px;
+  padding-top: 3em;
+  color: black;
+  position: relative;
+  bottom: 0;
+  background-color: #e4c1b1;
+}
+
+.row {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.list-footer li {
+  list-style: none;
+}
+.list-footer li a {
+  color: black;
+  text-decoration: none;
+  font-family: 'poppins', sans-serif;
+  font-size: 18px;
+  position: relative;
+  letter-spacing: 0.5px;
+  padding: 0 10px;
+}
+a:after {
+  content: '';
+  position: absolute;
+  background-color: rgb(0, 0, 0);
+  height: 3px;
+  width: 0;
+  left: 0;
+  bottom: -10px;
+  transition: 0.3s;
+}
+.list-footer li a:hover {
+  color: rgb(108, 3, 5);
+}
+a:hover:after {
+  width: 100%;
+}
+.main-sm-footer {
+  box-shadow: inset 0em 0.1em rgba(0, 0, 0, 0.2);
+  width: 1512px;
+  height: 64px;
+  background-color: rgba(228, 193, 177, 0.2);
+  color: black;
+}
+.list-footer {
+  padding-top: 10px;
+  line-height: 3;
+}
+.col,
+h4 {
+  padding-bottom: 20px;
+  font-size: large;
+}
+.icon {
+  height: 23px;
+  width: 33px;
+  border: 4px solid rgb(9, 9, 9);
+  border-radius: 50%;
+  background-color: rgb(252, 219, 219);
+  box-shadow: 0 10px 10px rgba(0, 0, 0, 0.1);
+  cursor: pointer;
```

Comment:
The border should not be there. The icons go in as they are. 
Pleas remove for the next iteration. 🙏🏽

Reply:
Okay 👍🏼